tests/experiments/order_classification_v6/order_classification_v6.py

```
@@ -0,0 +1,105 @@
+        llm=llm,
+        prompt=PromptTemplate(
+            input_variables=["query"],
+            template=PROMPT_TEMPLATE
+        )
+    )
+
+def get_order_class(
+    order_query: str,
+    rules_by_classes: dict,
+) -> dict:
+    llm = get_llm_by_client_credentials(client=client)
```

Ruff (0.8.2)

Undefined name `get_llm_by_client_credentials`

(F821)